ui/apps/everest/src/components/cluster-form/advanced-configuration/advanced-configuration.utils.ts

```diff
@@ -0,0 +1,58 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { DbType } from '@percona/types';
+import { DbCluster, ProxyExposeType } from 'shared-types/dbCluster.types';
+import { AdvancedConfigurationFields } from './advanced-configuration.types';
+import { AdvancedConfigurationFormType } from './advanced-configuration-schema';
+
+export const getParamsPlaceholderFromDbType = (dbType: DbType) => {
+  let dynamicText = '';
+
+  switch (dbType) {
+    case DbType.Mongo:
+      dynamicText = 'operationProfiling:\nmode: slowOp\nslowOpThresholdMs: 200';
+      break;
+    case DbType.Mysql:
+      dynamicText =
+        '[mysqld]\nkey_buffer_size=16M\nmax_allowed_packet=128M\nmax_connections=250';
+      break;
+    case DbType.Postresql:
+    default:
+      dynamicText =
+        'log_connections = yes\nsearch_path = \'"$user", public\'\nshared_buffers = 128MB';
+      break;
+  }
+
+  return dynamicText && `${dynamicText}`;
```

Comment:
don't quite understand this return statement, as it's unpredictable. 
If we have a default case, shouldn't we just return the variable?

Reply:
good catch! it was done before as it is (ui/apps/everest/src/pages/database-form/database-form-body/steps/advanced-configurations/advanced-configurations.utils.ts) but i think it's a good idea to simplify it, because in any case we will have a string value